recipe/2024-04-terraform-provider-secrets.md

```diff
@@ -0,0 +1,263 @@
+# Handling of Secrets Data for Terraform Providers
+
+* **Status**: Pending
+* **Author**: lakshmimsft
+
+## Overview
+
+We've described the design for support of multiple Terraform Providers in Radius in the following PR: [Design Document to support multiple Terraform Providers](https://github.com/radius-project/design-notes/pull/39/files) and implemented secret handling for Private Terraform modules [Ref PR link](https://github.com/radius-project/radius/pull/7306).
+This document  describes in detail the handling of secrets data between the Engine and Driver for Terraform Providers building on the above designs.
```

Comment:
This overview is hard to parse without jumping through multiple links. Can you instead summarize the purpose of this doc in words and then add links at the end for anyone interested?

Reply:
updated overview. pls take another look.